lib/playgroundbook_renderer/chapter_collator.rb

```diff
@@ -11,47 +11,29 @@ def initialize(page_writer = PageWriter.new, ui = Cork::Board.new)
       @ui = ui
     end
 
-    def collate!(chapter_name, chapter_file_contents, imports)
+    def collate!(chapter_name, parsed_chapter, imports)
```

Comment:
are these mutating functions? So far it doesn't look like it ( which is what is implied by the `!`) if not you can kill all the bangs.